atlasdb-feign/src/main/java/com/palantir/atlasdb/http/FeignOkHttpClients.java

```diff
@@ -89,17 +91,21 @@ private FeignOkHttpClients() {
      */
     public static <T> Client newOkHttpClient(
             Optional<SSLSocketFactory> sslSocketFactory,
+            Optional<ProxySelector> proxySelector,
             String userAgent) {
-        return new OkHttpClient(newRawOkHttpClient(sslSocketFactory, userAgent));
+        return new OkHttpClient(newRawOkHttpClient(sslSocketFactory, proxySelector, userAgent));
     }
 
     @VisibleForTesting
-    static okhttp3.OkHttpClient newRawOkHttpClient(Optional<SSLSocketFactory> sslSocketFactory,
+    static okhttp3.OkHttpClient newRawOkHttpClient(
```

Comment:
We need to change the `FeignOkHttpClientsTest` to supply an empty optional here too.

Reply:
done